runtime/doc/news.txt

```diff
@@ -222,6 +227,8 @@ LSP
   https://microsoft.github.io/language-server-protocol/specification/#textDocument_linkedEditingRange
 • Support for related documents in pull diagnostics:
   https://microsoft.github.io/language-server-protocol/specifications/specification-current/#relatedFullDocumentDiagnosticReport
+• |vim.lsp.buf.signature_help()| now supports displaying signature help
+  with no active parameter highlighted.
```

Comment:
It is so much easier for everyone if we just use the protocol name. Otherwise users have to piece together wth this means.
```suggestion
• |vim.lsp.buf.signature_help()| supports "noActiveParameterSupport".
```